streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -141,9 +141,19 @@ public void testGetConsumerConfigs() {
     @Test
     public void testGetGroupInstanceIdConfigs() {
         props.put(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG, "group-instance-id");
+        props.put(StreamsConfig.mainConsumerPrefix(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG), "group-instance-id-1");
+        props.put(StreamsConfig.restoreConsumerPrefix(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG), "group-instance-id-2");
+        props.put(StreamsConfig.globalConsumerPrefix(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG), "group-instance-id-3");
```

Comment:
Why do we need this 3 lines? Setting the config without prefix would set it for all consumers anyway?

If you want to test the overwrite with prefix, should it be it's own test?

Reply:
I want to check that no prefix would take effects here -- note I used different values for different overrides, so if it is violated we know which prefix introduced the violation.

Reply:
Ack